subsys/bluetooth/controller/ll_sw/ull_llcp_internal.h

```diff
@@ -1020,7 +1031,17 @@ static inline void pdu_encode_reject_ext_ind(struct pdu_data *pdu, uint8_t rejec
 	return ull_cp_priv_pdu_encode_reject_ext_ind(pdu, reject_opcode, error_code);
 }
 
+void ull_cp_priv_pdu_decode_reject_ext_ind(struct proc_ctx *ctx, struct pdu_data *pdu);
```

Comment:
See PR 35867 where we replace the ull_cp_priv prefix with llcp_
Up to you if we do the same change in this PR or in a separate one

Reply:
Changed to use prefix `llcp_`.